findimports.py

```diff
@@ -1087,6 +1089,8 @@ def main(argv=None):
     options.add_argument('-D', '--depth', type=int,
                          dest='max_depth',
                          help='import depth in ast tree. Default: no limit')
+    options.add_argument('--attr', type=str, nargs='*', dest='attributes',
+                         help='Add dot graph attributes. E.g. "rankdir=TB"')
```

Comment:
Hmm, this means you have to do `--attr k1=v1 k2=v2 k3=v3` to add multiple attributes, and you can't use `-attr` more than once.

What do you think about using `'-a', '--attr', type=str, nargs=1, action='append', dest='attributes'`?  Then you'd have to do `-a k1=v1 -a k2=v2 -a k3=v3`.

Good idea or bad idea?

Reply:
That is probably better. Just 1 issue. `-a` flag is already taken... :(

Reply:
What about -A?